projects/manager/src/app/permissions/permissions.component.ts

```diff
@@ -12,33 +25,86 @@ import {MatPaginator} from '@angular/material/paginator';
 })
 export class PermissionsComponent implements AfterViewInit, OnChanges {
     @Input()
-    resourceType!: string;
+    resourceType!: ResourceType;
```

Comment:
Afaik we said to add the mandatory flag (or how it is called) to the input for `!` types.

Reply:
I added the flag 5f3f7d5d3e2d563fae64e15c23f4a3536d78a2db